plugins/ReactNativeStyle/ReactNativeStyle.js

```diff
@@ -102,6 +102,10 @@ class NativeStyler extends React.Component {
   _handleStyleRename(oldName: string, newName: string, val: string | number) {
     var style = shallowClone(this.state.style);
     delete style[oldName];
+    if (newName === '') {
+      this.setState({style});
+      return;
```

Comment:
The backend script should included in react-native client, so you need to run `npm link <path-of-react-devtools-core>` in your react-native project. If you can't see any changes, just restart react-native packager.

Remember to use RN ^0.43, because v0.42 before are used the old [backend vendor](https://github.com/facebook/react-devtools/tree/master/packages/react-devtools-core/vendor).